app/modules/users/components/network_users_nav.svelte

```diff
@@ -0,0 +1,164 @@
+<script>
+  import { i18n, I18n } from '#user/lib/i18n'
+  import { icon, loadInternalLink } from '#lib/utils'
+  import Flash from '#lib/components/flash.svelte'
+  import UsersHomeSectionList from '#users/components/users_home_section_list.svelte'
+  import app from '#app/app'
+
+  let flash, users, groups
+
+  app.request('fetch:friends')
+    .then(() => {
+      users = app.users.filtered.friends().toJSON()
+    })
+    .catch(err => flash = err)
+
+  app.request('wait:for', 'groups')
+    .then(() => groups = app.groups.toJSON())
+    .catch(err => flash = err)
+
+  let showUsersMenu, showGroupsMenu
+</script>
+
+<div class="lists">
+  <div class="list-wrapper">
+    <div class="list-label-wrapper">
+      <h2 class="list-label">{I18n('friends')}</h2>
+      {#if !showUsersMenu}
+        <button
+          class="menu-toggler"
+          on:click={() => showUsersMenu = true}
+          aria-controls="userMenu"
+        >
+          {@html icon('plus')} {i18n('add friend')}
+        </button>
+      {/if}
+      <div id="userMenu" class="buttons">
+        {#if showUsersMenu}
+          <button on:click={app.Execute('show:invite:friend:by:email')}
+          >
+            {@html icon('envelope')}
+            {I18n('invite')}
+          </button>
+          <button on:click={app.Execute('show:users:search')}
+          >
+            {@html icon('search')}
+            {I18n('search')}
+          </button>
+          <button on:click={app.Execute('show:users:nearby')}
+          >
+            {@html icon('map-marker')}
+            {I18n('find on map')}
+          </button>
+        {/if}
+      </div>
+    </div>
+    {#if users}
+      <UsersHomeSectionList docs={users} type="users" />
+    {/if}
+  </div>
+
+  <div class="list-wrapper">
+    <div class="list-label-wrapper">
+      <h2 class="list-label">{I18n('groups')}</h2>
+      {#if !showGroupsMenu}
+        <button
+          class="menu-toggler"
+          on:click={() => showGroupsMenu = true}
+          aria-controls="groupMenu"
+        >
+          {@html icon('plus')} {i18n('add group')}
+        </button>
+      {/if}
+      <div class="buttons groupMenu">
+        {#if showGroupsMenu}
+          <button on:click={app.Execute('show:groups:search')}
+          >
+            {@html icon('search')}
+            {I18n('search')}
+          </button>
+          <button on:click={app.Execute('show:groups:nearby')}
```

Comment:
Im not sure of the step to reproduce, but when i m on the "friends & groups" tab and i click on "find on map" of the "
![2023-02-20_19-38](https://user-images.githubusercontent.com/5363918/220180358-62daddf8-a2a6-4595-90fe-757302479111.png)
groups" section, and then if i dezoom a lot (around zoom 3 i would say) then the following flash error appears:

![2023-02-20_19-38](https://user-images.githubusercontent.com/5363918/220180124-fc168154-3176-4f05-8dd3-28d733133e29.png)

note that this does not happen when click on find on map from the friends section

Reply:
Should have been fixed by 66044db3e and 54a3da145, which depend on server PR https://github.com/inventaire/inventaire/pull/672